app/models/item.rb

```diff
@@ -7,7 +7,7 @@ class Item < ApplicationRecord
   #                 :model, :brand, :name, :owner_id, :po_number,
   #                 :value, :grant, :purchase_amt, :purchase_date,
   #                 :sell_amt, :sell_date, :stock_number,
-  #                 :source, :status, :condition, :comments, :item_image,
+  #                 :vendor_id, :status, :condition, :comments, :item_image,
```

Comment:
due to the rails5 merge this needs to be updated to allow `vendor_id` be allowed through strong params - see https://github.com/ReadyResponder/ReadyResponder/blob/development/app/controllers/items_controller.rb#L61